cli/internal/cloudcmd/rollback.go

```diff
@@ -29,6 +29,8 @@ func rollbackOnError(ctx context.Context, w io.Writer, onErr *error, roll rollba
 	fmt.Fprintln(w, "Attempting to roll back.")
 	if err := roll.rollback(ctx); err != nil {
 		*onErr = multierr.Append(*onErr, fmt.Errorf("on rollback: %w", err)) // TODO: print the error, or return it?
+		fmt.Println("Automatic rollback failed.")
+		fmt.Println("Please manually clean up any resources on your CSP and try running 'constellation terminate' again.")
```

Comment:
```suggestion
		*onErr = multierr.Append(*onErr, fmt.Errorf("Rollback failed: %w", err)) // TODO: print the error, or return it?
		fmt.Fprintln(w, "Try running 'constellation terminate', or manually clean up dangling resources on your CSP.")
```
Line above "Automatic rollback failed" already tells the user roll back failed.
Since the rollback action is only run during `create` we should not tell users to run `terminate` "again"